cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -579,7 +579,7 @@ private String subQuery(String index,CQLTermNode node, Table correlation) throws
       if (isTermConstant) {
         String termString = "";
         if(isTermUUID) {
-          termString = "(" + term + ")::UUID";
+          termString = "('" + Cql2SqlUtil.cql2like(term) + "')::UUID";
```

Comment:
You can you use Cql2SqlUtil.cql2like only after a LIKE operator.
For quoting a SQL string you need to create a new method Cql2SqlUtil.quotedString(String)